deployment/terraform/modules/rotation_lambda/redis_rotation_handler.py

```
@@ -0,0 +1,240 @@
+        REPLICATION_GROUP_ID,
+    )
+    _wait_for_group_available()
+
+
+def _test_secret(arn: str, _token: str) -> None:
+    """Verify the AWSPENDING token allows a Redis PING."""
+    try:
+        import redis as redis_lib  # type: ignore[import]
+    except ImportError:
+        logger.warning(
+            "redis package not available in Lambda layer – skipping live PING test. "
```

Ruff (0.16.2)

[warning] String contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)